### ISSUE TITLE
I see that the global lru list "g_fcache_head" is not protected by locks, which may cause something wrong? 

### ISSUE THREAD

**Issue by melody97** [ut-osa/assise#8]
Do I ignore something?

**Comment by wreda** [comment]
You're correct. The LibFS LRU list should be protected by a lock since the linked list methods are not thread-safe. We didn't get around to adding this since the fcache is only used when reading from SSDs. A pull request would be welcome here!
